test/benchee/conversion/count_test.exs

```diff
@@ -1,58 +1,63 @@
 defmodule Benchee.Conversion.CountTest do
   use ExUnit.Case
+  alias Benchee.Conversion.Count
   import Benchee.Conversion.Count
   doctest Benchee.Conversion.Count
 
+  defp unit_for(unit_atom) do
+    Map.fetch! Count.units, unit_atom
+  end
```

Comment:
Yeah, I think the atoms should be the primary interface for humans. I was just noticing that the tests are speaking :)

So yes, maybe a public `unit_for/1` (or even `unit/1`)